packages/devextreme/js/__internal/viz/chart_components/m_base_chart.ts

```diff
@@ -1334,17 +1247,19 @@ export const BaseChart = BaseWidget.inherit({
   },
 
   _populateSeries(data) {
-    const that = this;
     const seriesBasis = [];
-    const incidentOccurred = that._incidentOccurred;
-    const seriesThemes = that._populateSeriesOptions(data);
+    const incidentOccurred = this._incidentOccurred;
+    const seriesThemes = this._populateSeriesOptions(data);
     let particularSeries;
     let disposeSeriesFamilies = false;
 
-    that.needToPopulateSeries = false;
+    this.needToPopulateSeries = false;
+
+    seriesThemes.forEach((theme) => {
+      const findSeries = (s) => s.name === theme.name
```

Comment:
do you like this short form? i mean why don't write full paramName? like (series) => series.name === theme.name

Reply:
We have too many places like this. I suggest take it into account in the future.  what do you think about my suggestion?

Reply:
np)